packages/timeline-state-resolver/src/devices/__tests__/telemetrics.spec.ts

```diff
@@ -0,0 +1,199 @@
+import { TelemetricsDevice } from '../telemetrics'
+import { DeviceOptionsTelemetrics, DeviceType, StatusCode, TimelineObjTelemetrics } from 'timeline-state-resolver-types'
+import { Socket } from 'net'
+// eslint-disable-next-line node/no-extraneous-import
+import { mocked } from 'ts-jest/utils'
+import { TimelineState } from 'superfly-timeline'
+import { ResolvedTimelineObjectInstance } from 'superfly-timeline/dist/api/api'
+import { DoOrderFunctionNothing } from '../../doOnTime'
+
+const SERVER_PORT = 5000
+const SERVER_HOST = '1.1.1.1'
+
+const MOCKED_SOCKET_CONNECT = jest.fn()
+const MOCKED_SOCKET_WRITE = jest.fn()
+const SOCKET_EVENTS: Map<string, (...args: any[]) => void> = new Map()
+
+jest.mock('net', () => {
+	return {
+		Socket: jest.fn().mockImplementation(() => {
+			return {
+				connect: MOCKED_SOCKET_CONNECT,
+				write: MOCKED_SOCKET_WRITE,
+				on: (event: string, listener: (...args: any[]) => void) => {
+					SOCKET_EVENTS.set(event, listener)
+				},
+			}
+		}),
+	}
+})
+
+jest.mock('../../doOnTime', () => {
+	return {
+		DoOnTime: jest.fn().mockImplementation(() => {
+			return {
+				queue: (_time: number, _queueId: string | undefined, fcn: DoOrderFunctionNothing) => {
+					fcn()
+				},
+				on: jest.fn(),
+				dispose: jest.fn(),
+			}
+		}),
+	}
+})
```

Comment:
It seems like this mock or something else in this file causes other tests to fail.